dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -170,6 +170,15 @@
 		<AD id="timestampFormat" name="%timestampFormat" description="%timestampFormat.desc" required="false" type="String" />
 		<!-- End VMM ported Entries -->	
 
+	<AD id="bindAuthMechanism" name="internal" description="internal use only" required="false" type="String">
```

Comment:
Just wonder why we do not use beta instead of internal? also, should we have default value "simple"

Reply:
I was going to do initial commit as internal and then when we had useable runtime code, promote to beta.

Reply:
I was also planning to wait on setting the default to simple until we GA. No default for internal/beta.

Reply:
Will update to beta when we've delivered the runtime code, will set the default when we GA.